.yamllint

```diff
@@ -5,6 +5,7 @@ rules:
   line-length:
     max: 150
     level: warning
+    ignore: "tests/assets/qrcode.yaml" # tokens are big
```

Comment:
I thought we had a fix for that

Reply:
hm, what could the fix be? Split the token in multiple lines? I'd rather keep it as one line for easier copying-pasting.

Reply:
I think there was a way to tell the linter about the lenght of the value, but maybe I'm imagining things here :P

Reply:
we do 2 lines above: `max: 150` but increasing that to fit the token would be equivalent to disabling the check :). Unless you mean we should override that rule only for that line. I don't know, let's leave it as it is. It's not something that would break anything anyway.

Reply:
yeah indeed, seems good for now to me